app/assets/stylesheets/components/_alert.scss

```diff
@@ -26,22 +25,22 @@ $ico-offset: -10px;
 
 .alert-notice,
 .alert-success {
-  background-color: #f0faed;
+  background-color: #ebfcef;
```

Comment:
These colors aren't being used elsewhere at this point so I think it's fine that they are one-offs in this case.  I did talk with Ryan yesterday and we tried to consolidate all of the colors into one color palette and I will likely do some naming/renaming in a future PR for that.